controllers/grid/articleGalleys/form/ArticleGalleyForm.inc.php

```diff
@@ -20,17 +20,22 @@ class ArticleGalleyForm extends Form {
 	/** @var the article */
 	var $_submission = null;
 
+	/** @var the article version */
+	var $_submissionRevision = null;
```

Comment:
it would be nice to correct the comments for the variables here: name type description